backend/src/api/routes/auth/index.routes.ts

```
@@ -695,6 +696,19 @@ router.post('/admin/sessions', (req: Request, res: Response, next: NextFunction)
   }
 });
 
+//GET admin Session
+router.get('/admin/sessions/current', verifyToken, async (req: AuthRequest, res: Response, next: NextFunction) => {
```

ESLint

[error] Replace `'/admin/sessions/current',·verifyToken,` with `⏎··'/admin/sessions/current',⏎··verifyToken,⏎·`

(prettier/prettier)